src/modules/settings/BlockedSkills.svelte

```diff
@@ -0,0 +1,226 @@
+<script>
+import getValueJSON from '../system/getValueJSON';
+import setValue from '../system/setValue';
+import {
+  checkSkill,
+  clearCheckedSkills,
+  getCheckedSkills,
+  submitSkillChanges,
+} from './blockedSkills';
+
+let blockedSkillLists = getValueJSON('blockedSkillLists');
+let newListName = '';
+let error = false;
+let listName = '';
+
+function checkForDuplicates(list) {
+  return blockedSkillLists.some((l) => {
+    if (l.name === list.name) {
+      return false;
+    }
+    if (l.skills.length !== list.skills.length) {
+      return false;
+    }
+    return l.skills.every((s, i) => s === list.skills[i]);
+  });
+}
+
+const loadListChecks = [
+  [
+    (list) => !list,
+    (list) => `Unable to find list "${list.name}". Try refreshing the page.`,
+  ],
+];
+
+const updateListChecks = [
```

Comment:
we could have a function that calls checkForErrors with this array
blockedSkillLists can be a parameter
blockedSkillLists would have to be an arg to checkForErrors
moved to import